tests/acceptance/pageObjects/filesPage.js

```diff
@@ -335,6 +338,33 @@ module.exports = {
       return this
         .waitForElementVisible('@restorePreviousVersion')
         .click('@restorePreviousVersion')
+    },
+    enterFilterInput: function (input) {
+      return this.initAjaxCounters()
+        .click('@filterListButton')
+        .waitForElementVisible('@filterTextField')
+        .clearValue('@filterTextField')
+        .setValue('@filterTextField', input)
+        .waitForOutstandingAjaxCalls()
+        .click('@filterListButton')
+    },
+    getAllFilesFolders: function () {
```

Comment:
a webdav request is not work of a page document
place it into the webdav helper